osd/unknown_failure.json

```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Cluster incident identified",
-    "description": "Your cluster is identified to have a general failure regarding the alert '${ALERT_NAME}'. The initial issue can be described as '${BRIEF_DESCRIPTION}'. A diagnosis of the issue is underway",
+    "description": "Your cluster is identified to have a general failure regarding the alert '${ALERT_NAME}'. The initial issue can be described as '${BRIEF_DESCRIPTION}'. A diagnosis of the issue is underway. If the cluster will keep alerting for the next 5 buisness days, we will put alerts for the cluster on hold until recieving a response from you",
```

Comment:
it's not clear what you want to say here.
my guess is that you want to say that if the cluster continues to fail for 5 consecutive business days and alert SRE then SRE will silence the cluster until remediation is done by the customer

having said that what is the point of saying this here knowing that you don't ask for any action to be taken by the CU and you are saying that a diagnosis is underway

Reply:
You are right..

This is pulled from multiple places and hopefully will ironed out soon